backend/app/api/endpoints/knowledge.py

```
@@ -15,6 +15,7 @@
 from sqlalchemy.orm import Session
 
 from app.api.dependencies import get_db
+from app.db.session import SessionLocal
 from app.core import security
 from app.db.session import SessionLocal
```

Ruff (0.14.10)

Redefinition of unused `SessionLocal` from line 18

Remove definition: `SessionLocal`

(F811)